tests/pytest_framework/functional_tests/logpath/test_flags_catch_all.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2015-2019 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+
+def write_message_with_fields(tc, file_source, hostname, program, message):
+    bsd_message = tc.new_log_message().hostname(hostname).program(program).message(message)
+    bsd_log = tc.format_as_bsd(bsd_message)
+    file_source.write_log(bsd_log)
+    return bsd_message
+
+
+def test_flags_catch_all(tc):
+    # Check the correct output if the logpath is the following
+    # log {
+    #     source(s_file);
+    #     log { destination(d_file3); };
+    # };
+    # log { destination(d_file4); flags(catch-all);};
+    # input logs:
+    # Oct 11 22:14:15 host-A app-A: message from host-A and app-A
+
+    config = tc.new_config()
+
+    config.create_global_options(keep_hostname="yes")
+
+    file_source = config.create_file_source(file_name="input.log")
+    source_group = config.create_source_group(file_source)
+
+    file_destination3 = config.create_file_destination(file_name="output3.log")
+    destination_group3 = config.create_destination_group(file_destination3)
+
+    file_destination4 = config.create_file_destination(file_name="output4.log")
+    destination_group4 = config.create_destination_group(file_destination4)
```

Comment:
Can you update the numbering too? Or I would even prefer using more talkative names: `source_group`, `file_destination`, `catchall_file_destination`.

Reply:
Thanks, ok. Will be updated.